minicluster/src/main/java/tachyon/underfs/UnderFileSystemCluster.java

```diff
@@ -51,6 +51,16 @@ public void run() {
   private static String sUfsClz;
 
   /**
+   * @return the existing underfs, throwing IllegalStateException if it hasn't been initialized yet
+   */
+  public static synchronized UnderFileSystemCluster get() {
```

Comment:
we don't currently have any unit tests for this class, this is a method is really simple, and the build is broken - it probably makes sense to add tests in another PR